src/authorizations/components/EditTokenOverlay.tsx

```diff
@@ -99,7 +93,13 @@ const EditTokenOverlay: FC<Props> = props => {
     }
 
     setPermissions(newPerms)
-  }
+  }, [props])
+
+  useEffect(() => {
+    if (_.isEmpty(permissions)) {
+      formatPermissions()
+    }
```

Comment:
Can we import `isEmpty` directly here? I think the existing page is importing the entire lodash library (_), which we don't need.